authentication/authentication-internal-service/src/main/resources/cert/mosipio.crt

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can we externalize the cert and key..?

Reply:
Till kernel externalizes this certificate, we cant externalize it